test/specs/mainnet/getAddresses/matic.spec.js

```diff
@@ -1,13 +1,16 @@
 import * as dotenv from 'dotenv';
 dotenv.config(); // init dotenv
 import { Factory, PrimeSdk } from '@etherspot/prime-sdk';
+import { EtherspotBundler } from 'ethers';
```

Comment:
Please see above. I am not sure how this will work?

Reply:
Its new implementation is available in the examples.

Reply:
Committed updated code